client/src/lib.rs

```diff
@@ -59,6 +59,8 @@ pub use engine::snapshot::{
 
 pub use engine::vault::RecordHint;
```

Comment:
```suggestion
pub use engine::{self, vault::RecordHint};
```
Just in case you didn't know the syntax :)

Reply:
Right, fixed.